gno.land/pkg/sdk/vm/handler.go

```diff
@@ -91,10 +106,10 @@ func (vh vmHandler) Query(ctx sdk.Context, req abci.RequestQuery) abci.ResponseQ
 		res = vh.queryRender(ctx, req)
 	case QueryFuncs:
 		res = vh.queryFuncs(ctx, req)
-	case QueryEval:
-		res = vh.queryEval(ctx, req)
 	case QueryFile:
 		res = vh.queryFile(ctx, req)
+	case QueryEval:
```

Comment:
nit: Why move?

Reply:
I'm not sure, ahah. I think I did this to match the order of the functions below